qemu/tests/cfg/win_irq_check.cfg

```diff
@@ -0,0 +1,22 @@
+- win_irq_check: single_driver_install.with_viorng single_driver_install.with_balloon
+    type = win_irq_check
+    only Windows
+    kill_vm = yes
+    login_timeout = 360
+    msinfo_cmd = start /wait msinfo32 /report c:\msinfo.txt /categories +resourcesirqs
+    irq_cmd = type c:\msinfo.txt |findstr /i /c:"%s" |findstr /i "^irq.*ok"
+    variants:
+        - with_balloon:
+            driver_name = balloon
+            device_name = "VirtIO Balloon Driver"
+            balloon = balloon0
+            balloon_dev_devid = balloon0
+            balloon_dev_add_bus = yes
+        - with_viorng:
+            driver_name = viorng
+            device_name = "VirtIO RNG Device"
+            no_virtio_rng:
+                virtio_rngs += " rng0"
+                backend_rng0 = rng-random
+                backend_type = passthrough
+                filename_passthrough = /dev/random
```

Comment:
why following three parameters are set?
backend_rng0 = rng-random
backend_type = passthrough
filename_passthrough = /dev/random 

Can we just add "no no_virtio_rng" here?

Reply:
No, I tried without that three paramters, the result prompt "ParamNotFound: Mandatory parameter 'backend' is missing"
And I create this cfg refering to single_dirver_install.cfg

Reply:
Ok.
Please change filename_passthrough to /dev/urandom, as all test cases now use /dev/urandom

Reply:
Ok, modified.